app/controllers/forest_liana/resources_controller.rb

```diff
@@ -141,7 +141,14 @@ def destroy
         checker = ForestLiana::PermissionsChecker.new(@resource, 'delete', @rendering_id)
         return head :forbidden unless checker.is_authorized?
 
-        @resource.destroy(params[:id]) if @resource.exists?(params[:id])
+        # NOTICE: destroy one or many records.
+        if (params[:id] && params.dig('data', 'attributes') == nil)
+          @resource.destroy(params[:id]) if @resource.exists?(params[:id])
+        else 
+          ids = ForestLiana::ResourcesGetter.get_ids_from_request(params)
+          @resource.destroy(ids)
+        end
```

Comment:
[Thinking] I wonder if it is a good idea to have 1 single controller for 2 routes.
I know that you'll factorize some logic but:
- it makes the code harder to read
- the `rescue` error log is the same for single and multiple destroy so the context will be hard to get in case of error.

Maybe `class ForestLiana::Router` should be adapted to create a dedicated controller for multiple destroy.